### PR TITLE
Removes tg

### PR DESCRIPTION
**Pull request by Archanial**
## About The Pull Request

Removes some mentions to tg and replaces them with bee

## Why It's Good For The Game

:crab: TG is gone :crab:

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -4,4 +4,4 @@ about: Suggest an idea for this project
 
 ---
 
-Feature requests are not handled in the repository. The best place to discuss these ideas would be on the /tg/station 13 forums here: https://tgstation13.org/phpBB/viewforum.php?f=9&sid=5153c1c704a4fb1006bf7a265e45e03f
+Feature requests are not handled in the repository. The best place to discuss these ideas would be on the beestation forums here: https://forums.beestation13.com/
```
Comment (St0rmC4st3r) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Except they are. They are and should be requested in a form of a github issue.

**Review by St0rmC4st3r — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (Archanial) [reply .github/ISSUE_TEMPLATE/feature_request.md]: If crossed wants to completly remove this I'll push the change, but it's like that for a really long time now.

Comment (Crossedfall) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Afaik we're fine with feature requests on GitHub. Any opinions, @ike709 ?

**Comment by ghost** [comment]
No

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (ike709) [reply .github/ISSUE_TEMPLATE/feature_request.md]: github is fine

**Comment by Moccha-Bee** [comment]
> No

/TG/ player above

**Comment by Chessus** [comment]
There's also code that plays that "Now approaching tg station". character.playsound_local(get_turf(character), 'sound/voice/ApproachingTG.ogg', 25). it's here code/modules/mob/dead/new_player/new_player.dm

**Comment by Archanial** [comment]
I mean yeah if you want to record your own version that says beestation and sounds similar feel free to dm me on discord

**Comment by ghost** [comment]
> > No
> 
> /TG/ player above

Yes

**Comment by Archanial** [comment]
> 
> 
> > > No
> > 
> > 
> > /TG/ player above
> 
> Yes

:vomiting_face:

**Comment by Moccha-Bee** [comment]
> > > No
> > 
> > 
> > /TG/ player above
> 
> Yes

Can we please ban him from our repo/discord/server ? thank you

**Comment by ghost** [comment]
> > > > No
> > > 
> > > 
> > > /TG/ player above
> > 
> > 
> > Yes
> 
> Can we please ban him from our repo/discord/server ? thank you

Jeez thanks, so kind. Listen man, /tg/ banned me from their github, so I’m just trying to find a new place to chill. This is simply my heavy vetting process. You are getting low marks right now for this

**Comment by Moccha-Bee** [comment]
> > > > > No
> > > > 
> > > > 
> > > > /TG/ player above
> > > 
> > > 
> > > Yes
> > 
> > 
> > Can we please ban him from our repo/discord/server ? thank you
> 
> Jeez thanks, so kind. Listen man, /tg/ banned me from their github, so I’m just trying to find a new place to chill. This is simply my heavy vetting process. You are getting low marks right now for this

If tg repo banned you then why are you defending them ???????

**Comment by ghost** [comment]
Life’s too short to hold a grudge, and what I did was wrong and I deserved it

**Comment by ghost** [comment]
> Life’s too short to hold a grudge, and what I did was wrong and I deserved it

bee is reddit station, don't stay here for your own sake.

**Comment by Moccha-Bee** [comment]
> > Life’s too short to hold a grudge, and what I did was wrong and I deserved it
> 
> bee is reddit station, don't stay here for your own sake.

Ignore belony he is absolutly bolonies, beestation is based and red pilled

**Comment by ghost** [comment]
You are some weird people. /tg/ is infested with Redditors aswell, hard to avoid at this point

**Comment by ghost** [comment]
solution: stop playing this awfull game.

**Comment by ghost** [comment]
And do what?

**Comment by ghost** [comment]
> And do what?

play good games

**Comment by ghost** [comment]
I am poor tho

**Comment by ghost** [comment]
> I am poor tho

solution: get a job.

**Pull request merged**